cypress.config.ts

```diff
@@ -0,0 +1,17 @@
+import { defineConfig } from "cypress";
+
+export default defineConfig({
+  projectId: "h1jieb",
+  video: true,
```

Comment:
Should default video to false?

Reply:
is there a way to get video in the dashboard if defaulted to false?

Reply:
Ah if we're planning on uploading the videos we could keep this option false and add a --record cli option when running the tests in the ci